CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+- Add `dataclass` support for `pytorch_lightning.utilities.apply_func` ([#7935](https://github.com/PyTorchLightning/pytorch-lightning/pull/7935))
```

Comment:
```suggestion
- Add `dataclass` support for `pytorch_lightning.utilities.apply_to_collection` ([#7935](https://github.com/PyTorchLightning/pytorch-lightning/pull/7935))
```
I think users will recognize `apply_to_collection` better when reading the changelog so I suggest this